src/Phinx/Migration/Manager.php

```diff
@@ -206,8 +206,8 @@ public function printStatus($environment, $format = null)
                     '%s %14.0f  %19s  %19s  <comment>%s</comment>',
                     $status,
                     $migration->getVersion(),
-                    $version['start_time'],
-                    $version['end_time'],
+                    ($version['start_time'] ?? ''),
```

Comment:
I think the previous change was cleaner, as accessing this as array if bool is still wrong this way IMO.

Reply:
Also please note https://github.com/cakephp/phinx/blob/master/composer.json#L29 - as such this syntax is invalid.

Reply:
reverted to previous version

Reply:
Notice that strangely https://github.com/cakephp/phinx/blob/0.11.x/composer.json#L28